x-pack/plugins/rollup/public/crud_app/sections/job_create/steps_config/validate_rollup_index.js

```diff
@@ -28,13 +27,7 @@ export function validateRollupIndex(rollupIndex, indexPattern) {
     )];
   }
 
-  const illegalCharacters = INDEX_ILLEGAL_CHARACTERS_VISIBLE.reduce((chars, char) => {
-    if (rollupIndex.includes(char)) {
-      chars.push(char);
-    }
-
-    return chars;
-  }, []);
+  const illegalCharacters = findIllegalCharactersInIndexName(rollupIndex);
```

Comment:
noticed there was an existing util that was already doing the same logic so updated to use it here

Reply:
good find! this was a new util we added for CCR, nice to consolidate it here 😄